bears/rest/rstcheckBear.py

```diff
@@ -0,0 +1,51 @@
+from rstcheck import check
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Result import Result
+
+
+class rstcheckBear(LocalBear):
+    LANGUAGES = {"reStructuredText"}
+    REQUIREMENTS = {PipRequirement('rstcheck')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax'}
+    ASCIINEMA_URL = 'https://asciinema.org/a/ccjtkotaqzgggkkqtgelmj02x'
+
+    def run(self, filename, file, code_block_language_ignore: str,
+            report: str):
+        """
+        A Bear for rstcheck (https://pypi.python.org/pypi/rstcheck)
+
+        rstcheck checks syntax of reStructuredText and code blocks
+        nested within it
+        supports bash, c, cpp, json, python and rst
+
+        :param code_block_language_ignore: comma seperated list for languages
+         of which code blocks should be ignored (bash,c,cpp,json,python,rst)
+
+        :param report: report system messages at or higher than level;
```

Comment:
I think we do not want a "report" parameter.

Instead, we want to capture all messages from `rstcheck`, and filter them in coala using the coala severity options.

To do this, we need to extract the error level from each message.

i.e. when rstcheck prints out:

```
README.rst:175: (INFO/1) Duplicate implicit target name: "authors".
```

We need to extract "INFO" and covert it to the coala severity.

Coala automatically maps most words.  If rstcheck uses words that coala doesnt understand, a custom severity map can be created: http://docs.coala.io/en/latest/Developers/Writing_Linter_Bears.html#using-severities